### DISCUSSION TITLE
VTE improvement

### DISCUSSION THREAD

**Discussion by wsalmeida** [geany/geany#2873] [Ideas]
When you are editing something inside the VTE I have the impression that it steals the focus and I can't run some shortcuts causing discomfort for those who use shortcut keys for actions in the IDE my suggestion would be how not to let it not happen when I'm in the VTE?

**Comment by elextr** [comment]
It is deliberate that only a very few Geany shortcuts work in the VTE.

Because Geany does not know what program you are running in the VTE and what keybindings that program might need, so few are "stolen" from it.

**Comment by wsalmeida** [comment]
It makes sense to have right then.

Em sáb., 21 de ago. de 2021 22:08, elextr ***@***.***>
escreveu:
